### DISCUSSION TITLE
MTV HD | (Russia)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#348] [Show and tell]
### Channel Name

MTV HD

### Channel ID

MTVRussia.ru

### Website

http://myott.top/

### Stream URL

http://myott.top/stream/1VGPHYLZTH/89.m3u8

### Notes

Here Are Some Other MTV Channels 

MTV 80s
http://myott.top/stream/1VGPHYLZTH/87.m3u8

MTV 90s
http://myott.top/stream/1VGPHYLZTH/88.m3u8

MTV 00s
http://myott.top/stream/1VGPHYLZTH/165.m3u8

MTV Live
http://myott.top/stream/1VGPHYLZTH/215.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
http://myott.top/stream/1VGPHYLZTH/89.m3u8 (offline)
<img width="1060" alt="image" src="https://user-images.githubusercontent.com/7253922/232226952-74eea7c5-ec58-47b7-8028-a3e4bf0f76ed.png">
